modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -1,13 +1,15 @@
 // Module included in the following assemblies:
 //
 // * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
 :_content-type: REFERENCE
 [id="rosa-cluster-autoscale-settings_{context}"]
 = Cluster autoscaling settings using OpenShift Cluster Manager
 
 The tables explain all the configurable UI settings when using cluster autoscaling with OpenShift Cluster Manager.
 
-== Balancing behavior
+== General settings
 
```

Comment:
Table needs a title. Something like?..
.Configurable settings for cluster autoscaling when using the OpenShift Cluster Manager UI